### PR TITLE
docs: update content and images

### PR DESCRIPTION
**Pull request by NickIliev**
related to https://kinvey.atlassian.net/browse/FID-4749 
https://kinvey.atlassian.net/browse/FID-4748
https://kinvey.atlassian.net/browse/FID-4745
https://kinvey.atlassian.net/browse/FID-4741

+ updating images

### CODE REVIEW

File: knowledge-base/live-traffic-pane.md [hunk 1]
```diff
@@ -450,28 +450,44 @@ The column **Filter** menu lets you add filters for the currently selected colum
...
 
+### Adding New Rules
+
+You can use a captured session entry to quickly mock client or server behavior. For similar cases, you can use the session URL alongside the [**Rules** tab]({%slug modify-traffic-get-started%}). Creating rules allows you to test complex scenarios and various mock responses.
+
+To add a new rule:
+
+1. Select the desired session entry and right-click to open the context menu. Select **Add New Rule**.
+
+1. A new rule is automatically created. The rule will use the URL from the selected session entry and, by default, will apply the initial response through the **Manual Response** action option.
```
Comment (jivkopiskov) [inline knowledge-base/live-traffic-pane.md:487]: shoudln't this be 2?

File: knowledge-base/live-traffic-pane.md [hunk 2]
```diff
@@ -497,6 +513,17 @@ All options will display a **Save** prompt window. To save the desired sessions:
 Alternatively, you can save all captured traffic through the **Save** button from the toolbar.
 
 
+### Exporting
```
Comment (jivkopiskov) [inline knowledge-base/live-traffic-pane.md:516]: We will only work with Selected sessions now, there are no dropdowns with selected/unselected/all sessions

File: knowledge-base/live-traffic-pane.md [hunk 3]
```diff
@@ -576,24 +586,15 @@ To select sessions directly related to the currently selected session, use the *
...
+### Marking
 
-- **Copy as Fetch**&mdash;converts the request as a Fetch code and puts it in the system clipboard.
+Use the **Mark** context menu option to apply colors or strike out single or multiple sessions.
 
-- **Copy as Curl**&mdash;converts the request as a Curl command and puts it in the system clipboard.
+1. Select the sessions, open the context menu, and click the **Mark** option.
 
-- **Copy as Python (requests)**&mdash;converts the request as a Python code and puts it in the system clipboard.
+1. Click the preferred marking style to apply it.
```
Comment (jivkopiskov) [inline knowledge-base/live-traffic-pane.md:595]: maybe it's just how numbering in this editor is done, but is it supposed to say 1 on both lines here as well?

File: images/settings/settings-https.png
Comment (jivkopiskov) [inline images/settings/settings-https.png:1]: I see this file has been changed, but there is no mention that the new advanced option (install in machine store) is only available on Windows

**Review by jivkopiskov — APPROVED** [review]

File: knowledge-base/live-traffic-pane.md (hunk 3 above)
Comment (NickIliev) [reply knowledge-base/live-traffic-pane.md:595]: that's fine - it's a markdown thingy to create numbered lists (without actually worrying about the numbers)

**Pull request merged**